examples/testapp/vitest.config.ts

```diff
@@ -0,0 +1,13 @@
+import path from 'node:path';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    alias: {
+      ':core': path.resolve(__dirname, 'src/core'),
+      ':util': path.resolve(__dirname, 'src/util'),
+    },
```

Comment:
We can remove these aliases

Reply:
[7e8377b](https://github.com/coinbase/coinbase-wallet-sdk/pull/1489/commits/7e8377bea347bb7b11e7f4f065285e4dd7fade16)